target/linux/bcm5862x/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+case "$(board_name)" in
+meraki,mx64)
+	ucidef_set_led_default "running" "Running LED" "power:white" "1"
+	ucidef_set_led_default "boot" "Boot LED" "power:orange" "0"
+	ucidef_set_led_netdev "wan0" "WAN Port" "wan0:right" "wan0" "link tx rx"
+	ucidef_set_led_netdev "lan1" "Port 1" "lan1:right" "lan1" "link tx rx"
+	ucidef_set_led_netdev "lan2" "Port 2" "lan2:right" "lan2" "link tx rx"
+	ucidef_set_led_netdev "lan3" "Port 3" "lan3:right" "lan3" "link tx rx"
+	ucidef_set_led_netdev "lan4" "Port 4" "lan4:right" "lan4" "link tx rx"
```

Comment:
"link tx rx" is default and can be omitted.

"lanX:right" is not the right syntax, this should be "color:function"